src/Okta.Sdk.UnitTests/Internal/MockedCollectionRequestExecutor{T}.cs

```diff
@@ -13,7 +13,7 @@
 
 namespace Okta.Sdk.UnitTests.Internal
 {
-    public class MockedCollectionRequestExecutor<T> : IRequestExecutor
+    public class MockedCollectionRequestExecutor<T> : DefaultRequestExecutor
```

Comment:
As discussed, let's implement the interface here and use `DefaultRequestExecutor` as an internal property.

Reply:
Per further discussion, the final implementation of this test won't be able to utilize the specific approach you mention; I did change this back to implementing the interface vs extending the base class.